antora-ui-camel/src/css/frontpage.css

```diff
@@ -207,7 +207,7 @@ section.frontpage h1 {
 }
 
 section.frontpage h2 {
-  text-align:center;
+  text-align: center;
```

Comment:
This is unrelated to the objective of my PR. But I had to make this change because yarn build was throwing an error.

Reply:
Yeah I've noticed that, fixed on master in 3a11366d3d70577c971bc151deab9dd1fbfea51b